code/modules/mob/living/carbon/xenomorph/evolution.dm

```diff
@@ -188,14 +188,18 @@
 		if(length(hive.xenos_by_typepath[/mob/living/carbon/xenomorph/shrike]))
 			to_chat(src, "<span class='warning'>There already is a living Shrike. The hive cannot contain more than one psychic energy repository.</span>")
 			return
-		
+
 		if(mind)
 			mind.assigned_role = ROLE_XENO_QUEEN
 
 	else
 		var/potential_queens = length(hive.xenos_by_typepath[/mob/living/carbon/xenomorph/larva]) + length(hive.xenos_by_typepath[/mob/living/carbon/xenomorph/drone])
 
-		tierones = length(hive.xenos_by_tier[XENO_TIER_ONE])
+		if(istype(hive, /datum/hive_status/normal)) //If it has burrowed larva let's include that
+			var/datum/hive_status/normal/hive2
+			tierones = length(hive2.xenos_by_tier[XENO_TIER_ONE]) + length(hive2.xenos_by_tier[XENO_TIER_ZERO]) + length(hive2.stored_larva) //Includes larva and burrowed larva
+		else
+			tierones = length(hive.xenos_by_tier[XENO_TIER_ONE]) + length(hive.xenos_by_tier[XENO_TIER_ZERO])
```

Comment:
```suggestion
			tierones = length(hive.xenos_by_tier[XENO_TIER_ONE]) + length(hive.xenos_by_tier[XENO_TIER_ZERO])
			if(istype(hive, /datum/hive_status/normal))
							var/datum/hive_status/normal/normal_hive = hive
							tierones += normal_hive.stored_larva
```

* `var/datum/hive_status/normal/hive2` is undefined, but a more descriptive name like `normal_hive` is preferable anyway.
* `stored_larva` is not a list, it's already a numeric value.